libs/sdk-ui-filters/src/AttributeFilterLoader/internal/store/attributeElements/sagas.ts

```diff
@@ -14,33 +15,37 @@ export function* attributeElementsWorker(): SagaIterator<void> {
 }
 
 function* attributeElementsRequestSaga({
-    payload: { correlationId, displayFormRef, elements, limit, offset },
+    payload: { correlationId, ...options },
 }: ReturnType<typeof actions.attributeElementsRequest>): SagaIterator<void> {
     const context: SagaReturnType<typeof getAttributeFilterContext> = yield call(getAttributeFilterContext);
-    yield call(() =>
-        cancelableCall({
-            promise: () =>
-                loadAttributeElements(context, {
-                    displayFormRef,
-                    elements,
-                    limit,
-                    offset,
-                }),
-            onSuccess: function* (attributeElementsResult) {
-                yield put(
-                    actions.attributeElementsSuccess({
-                        correlationId,
-                        attributeElements: attributeElementsResult.items,
-                        totalCount: attributeElementsResult.totalCount,
-                    }),
-                );
-            },
-            onError: function* (error) {
-                yield put(actions.attributeElementsError({ error, correlationId }));
-            },
-            onCancel: function* () {
-                yield put(actions.attributeElementsCancel({ correlationId }));
-            },
-        }),
+    const displayFormRef: ReturnType<typeof selectAttributeFilterDisplayForm> = yield select(
+        selectAttributeFilterDisplayForm,
     );
+
+    const cancelableElementsLoad = cancelableEffect({
+        effect: () =>
+            loadAttributeElements(context, {
+                ...options,
+                displayFormRef,
+            }),
+        isCancelRequest: actions.attributeElementsCancelRequest.match,
+    });
+
+    const { success, error, cancel }: SagaReturnType<typeof cancelableElementsLoad> = yield call(
```

Comment:
Nitpick: should the `cancel` be called `cancelled`? Because at first glance `cancel` looks like something I can call to cancel the call.